changelog/2022-01-21-conduit-0-1-0-release.md

```diff
@@ -0,0 +1,38 @@
+---
+slug: '2022-01-21-conduit-0-1-0-release'
+title: Conduit 0.1.0 release
+draft: false
+tags: [conduit, release]
+---
+
+[**Conduit** is now open-sourced](https://github.com/ConduitIO/conduit/releases/tag/v0.1.0)! Designed to simplify data integration for software engineers, Conduit provides a flexible and developer-friendly solution for streaming data orchestration.
+
+<!--truncate-->
+
+### Key Features
+
+- **Easy Setup**: Get started quickly by downloading the Conduit binary, unzipping it, and building pipelines. Simply run the following commands:
+
+```bash
+$ tar zxvf conduit_0.1.0_Darwin_x86_64.tar.gz
+$ ./conduit
+```
+
+- **User Interface**: Conduit includes a built-in UI for local development, allowing you to easily navigate to [http://localhost:8080/ui/](http://localhost:8080/ui/) and start building pipelines.
+- **Developer-Centric Design**: Conduit aims to provide an excellent experience for connector developers, ensuring consistency and familiarity with modern programming languages.
```

Comment:
What does "familiarity with modern programming languages" mean in this context?

Reply:
I think "languages" is an oversight. What I meant is "familiarity with modern programming practices". I'll change, but let me know what you think.